.github/workflows/validate-pr.yml

```diff
@@ -3,7 +3,8 @@ name: Validate PR
 on:
   workflow_dispatch:
   pull_request:
-    types: [auto_merge_enabled]
+    types: [opened, synchronize, reopened, ready_for_review, auto_merge_enabled]
```

Comment:
Defaults are "opened, synchronized and reopened".
Adding additionally ready_for_review would mean it runs twice. After you pushed (synchronize), then you set it to ready for review (if it was a draft). Is das correct? 🤔
If you got your review and set the PR to auto_merge_enabled it would even run a third time?

I ask because I wanted to know
- when ready for review and auto merge enabled got triggered
- if just pull_request without any types would be enough 🤷‍♂️

Reply:
Yeap, you're right
I thought that actions wouldn't rerun a pr without changes